lib/wayland-glib/wl-source.vala

```diff
@@ -0,0 +1,44 @@
+namespace WlSource {
```

Comment:
Kind of dislike the namespace name
How about `namespace WlGlib` since the lib is called wayland-glib?